hardware-memory-isolation/activities/read-gdt/public/Makefile

```diff
@@ -0,0 +1,7 @@
+obj-m += read_gdt.o
+
+all:
+	make -C /home/cristi/WSL2-Linux-Kernel M=$(shell pwd) modules
+
+clean:
+	make -C /home/cristi/WSL2-Linux-Kernel M=$(shell pwd) clean
```

Comment:
```suggestion
all:
	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) modules

clean:
	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) clean
```

Reply:
Besides this, ask people somewhere in the session to install the kernel header files.
```console
$ apt-cache search linux-headers-`uname -r`
$ sudo apt install linux-headers.....
```

Also ask them to run this from inside a virtual machine.